include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -308,7 +308,7 @@ struct OrtCustomOp;
 typedef struct OrtCustomOp OrtCustomOp;
 
 typedef enum OrtAllocatorType {
-  Invalid = -1,
+  OrtInvalidAllocator = -1,
```

Comment:
If a user includes the new header in her old code, the build might fail. So these are source incompatible changes. We should call this out somewhere (may be in the release notes).

Reply:
Yes I made sure to keep it binary compatible but it's definitely not source compatible.